setup.py

```diff
@@ -304,6 +304,8 @@ def init_settings_from_config(self):
             pjoin('backend', 'cython'),
             'devices',
         )]
+        if sys.platform.startswith('win'):
+            settings['include_dirs'].insert(0, 'patch')
```

Comment:
I just checked and it seems `zmq.h` in 4.0.x and 4.1.x provide their own definitions of integer types but master does not.  See:
- [zmq.h line 73 in zeromq/libzmq](https://github.com/zeromq/libzmq/blob/master/include/zmq.h#L73) -- no integer types,
- [zmq.h line 73 in zeromq/zeromq4-x](https://github.com/zeromq/zeromq4-x/blob/master/include/zmq.h#L73) -- integer types; and
- [zmq.h line 73 in zeromq/zeromq4-1](https://github.com/zeromq/zeromq4-1/blob/master/include/zmq.h#L73) -- integer types.

I'm guessing master is "wrong" here, but the stable fork's definitions are kind of strange (is it even legal C to use preprocessor directives to see if a `typedef` has been defined?)